app/controllers/walks_controller.rb

```diff
@@ -14,7 +14,8 @@ def create
     @walk = Walk.new(walks_params)
     @walk.requester_id = current_user.id
 
-    if @walk.save
+    if @walk.save 
+      @walk.accepted = true      
```

Comment:
This line doesn't do anything, but it seems like this PR is missing the callback function inside the walk model.